### PR TITLE
Service scopes and FAQs for...

### PR DESCRIPTION
**Pull request by suehighmoor**
CyberScore, Moogsoft and AppDynamics

### CODE REVIEW

File: articles/third-party/third-faq-appdynamics.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+toc_sub3:
+toc_sub4:
+toc_title: Cisco AppDynamics from UKCloud FAQs
+toc_fullpath: Cisco AppDynamics from UKCloud/third-faq-appdynamics.md
+toc_mdlink: third-faq-appdynamics.md
+---
+
+# Cisco AppDynamics from UKCloud FAQs
+
+AppDynamics provides Getting Started guides, Troubleshooting, Documentation, Community support and a Knowledge Base which you can found in the [Support section of the AppDynamics website](https://www.appdynamics.com/support/).
```
Comment (SteveHallUKCLoud) [inline articles/third-party/third-faq-appdynamics.md]: should be "find"

File: articles/third-party/third-faq-cyberscore.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+# XQ Cyber CyberScore from UKCloud FAQs
+
+<sup>*</sup> All links to CyberScore support documents will require you to log in.
+
+## General
+
+### What is CyberScore?
+
+CyberScore&trade; is a fully-automated security testing and rating service that scores a virtual machine’s cyber security posture and explains potential impacts. Results are presented in a range of reports and action plans that help customers rank and rate vulnerabilities and identify key remediations. You can run the service as often as required and will CyberScore will show your progress over time.
```
Comment (SteveHallUKCLoud) [inline articles/third-party/third-faq-cyberscore.md]: duplicate and rebellious "will"

File: articles/third-party/third-faq-appdynamics.md (hunk 1 above)
Comment (suehighmoor) [reply articles/third-party/third-faq-appdynamics.md]: Fixed

File: articles/third-party/third-faq-cyberscore.md (hunk 2 above)
Comment (suehighmoor) [reply articles/third-party/third-faq-cyberscore.md]: Fixed

**Comment by cam-ukcloud** [comment]
Should the links to UKCloud Ideas be changed to Communities?

**Comment by cam-ukcloud** [comment]
Have made some changes to the CyberScore FAQ doc.

**Comment by suehighmoor** [comment]
Made a couple of minor changes and updated the Feedback section

**Pull request merged**